src/container/keys.ts

```diff
@@ -1,4 +1,5 @@
 export const keys = {
+  initializing: "_initializing",
```

Comment:
@kevinwonart I have a task on the backlog that I think will be a good first one for you. It is going to use this key and the `isInitializing` function below